### PR TITLE
(DOCS-2057) Teams and Faces updates

### PR DESCRIPTION
**Pull request by jtappa**
## Motivation

The "Teams" page no longer exists. Personal Settings and Organization Settings have been totally revamped, they have a different left nav now rather than the top nav they previously had. Everything that used to live at **Integrations -> APIs** has moved to organization settings and personal settings, so all mentions of these things being in Integrations needed to be updated. (There will continue to be a page that says these things have moved, but we still want to direct users to the new correct location.)

DOCS-2057

### Preview

https://docs-staging.datadoghq.com/jorie/DOCS-2057/account_management/
https://docs-staging.datadoghq.com/jorie/DOCS-2057/account_management/org_settings/
https://docs-staging.datadoghq.com/jorie/DOCS-2057/account_management/api-app-keys/
https://docs-staging.datadoghq.com/jorie/DOCS-2057/account_management/org_switching/
https://docs-staging.datadoghq.com/jorie/DOCS-2057/account_management/plan_and_usage/
https://docs-staging.datadoghq.com/jorie/DOCS-2057/account_management/users/
https://docs-staging.datadoghq.com/jorie/DOCS-2057/events/guides/email/

---

### Reviewer checklist (Updated specific to this PR)
- [ ] Ensure any images removed from markdown files have also had the image files removed in the files changed section.
- [ ] Review all changed pages in the Preview section above.
  - [ ] View the staging preview to make sure any images with the little teams icon have been removed or cropped. 
  - [ ] View the left nav under account management, and ensure it follows this format:

  * Switching between orgs
  * Organization Settings
     * User management
  * RBAC
      * Permissions
  * SSO with SAML
  * API and Application Keys
  * Plan and Usage
  * Billing
  * Multi-org accounts
 

### To-Do (Jorie)
- [x] /account_management/ index updates
- [x] /account_management/org_settings updates
- [x] /account_management/users/ updates
- [x] /events/guides/email/ updates - (update directions)
- [x] account_management/org_switching/ updates
- [x] /account_management/api-app-keys/ updates
- [x] re-org side nav to have API and Application keys higher in the list, and add the plan and usage page
- [x] Remove any screenshots that have the "Teams" icon 
- [x] Remove or update any screenshots that have the old user/org settings links
- [x] Make sure to `git -rm` the actual files for images removed from markdown files

### CODE REVIEW

**Comment by jtappa** [comment]
Note for Kaylyn:
The only thing I didn't get fixed up was the sidenav, it was just too hairy with conflicts and gross and I got frustrated and gave up 😭 sorry. But I added what I was *trying* to get it to look like in the review checklist, if you can take a stab at it. There's just one new page, and it's definitely not *required* to re-org all the links around, I just thought this layout would make more sense based on how things are organized in the app now.

**Comment by jtappa** [comment]
Update: Giving the sidebar another try this afternoon. Will change PR from Draft once those commits are in place.

File: content/en/events/guides/email.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ If your application does not have an existing [Datadog integration][1], and you
 Before you can send events with email, you need a dedicated email address from Datadog:
 
 1. Log in to your [Datadog account][3].
-2. Go to the **Integrations** menu, choose **APIs**.
-3. Expand **Events API emails**.
+2. From the Account menu at the bottom left, select **Organization Settings**
```
Comment (alai97) [inline content/en/events/guides/email.md:16]:
```suggestion
2. From the **Account** menu at the bottom left, select **Organization Settings**.
```

File: content/en/account_management/users/_index.md [hunk 2]
```diff
@@ -22,61 +22,50 @@ further_reading:
...
 2. Select the *Edit* button on the right of the user line.
-3. Select the new [user roles][2] for this user.
-4. **Save** the new setting.
+3. Select the new [user roles][2] for this user, or click the 'X' next to an existing role to remove it.
+4. **Save** the new settings.
 
 {{< img src="account_management/users/user_role_update.png" alt="User role update" style="width:80%;">}}
 
-See the [Role Based Access Control][2] documentation to discover all roles available and how to create custom ones.
+See the [Role Based Access Control][2] documentation to discover all of the roles available and how to create custom ones.
```
Comment (alai97) [inline content/en/account_management/users/_index.md:56]:
```suggestion
To discover all of the roles available and how to create custom ones, see the [Role Based Access Control documentation][2].
```

File: content/en/account_management/users/_index.md [hunk 3]
```diff
@@ -22,61 +22,50 @@ further_reading:
...
 
 ## Edit a user's roles
 
-Only users with the Access Management permission, such as users with the Datadog Admin Role, can change another user's role:
+Only users with the Access Management permission, such as users with the Datadog Admin Role, can change another user's role.
+
+To edit a user's roles:
 
-1. Go to the User Management Page.
+1. Go to the Users tab of Organization Settings.
```
Comment (alai97) [inline content/en/account_management/users/_index.md:49]:
```suggestion
1. Go to the **Users** tab of **Organization Settings**.
```

File: content/en/account_management/users/_index.md [hunk 4]
```diff
@@ -22,61 +22,50 @@ further_reading:
...
 2. Select the *Edit* button on the right of the user line.
```
Comment (alai97) [inline content/en/account_management/users/_index.md:50]:
```suggestion
2. Select the **Edit** button on the right of the user line.
```

File: content/en/account_management/users/_index.md [hunk 5]
```diff
@@ -22,61 +22,50 @@ further_reading:
...
-The new user receives an email with a link to log in. The user is marked with status `Invite Pending` until they log in. To cancel their invite before they log in, click the *Delete Invite* button on the right of the user line in list view, or on the user box in grid view. 
-
-{{< img src="account_management/users/delete_invite_grid.png" alt="Delete invite on the grid view" style="width:50%;">}}
+5. Click **Send Invites**.
 
-To resend an invite in list view, click the user to open the user side panel, and click **Resend Invite**:
+The new user receives an email with a link to log in. The user is marked with the status `Invite Pending` until they log in. To cancel their invite before they log in, click the *Delete Invite* button on the right of the user line in list view, or on the user box in grid view. 
 
-{{< img src="account_management/users/resend_invite_list.png" alt="Resend invite on the list view" style="width:80%;">}}
+To resend an invite in list view, click the user to open the user side panel, and click **Resend Invite**. Or in grid view, hover over the user box and click **Resend Invite**.
```
Comment (alai97) [inline content/en/account_management/users/_index.md:40]:
```suggestion
To resend an invite in list view, click the user to open the user side panel and click **Resend Invite**. Or in grid view, hover over the user box and click **Resend Invite**.
```

File: content/en/account_management/users/_index.md [hunk 6]
```diff
@@ -22,61 +22,50 @@ further_reading:
...
-
-{{< img src="account_management/users/invite_user.png" alt="Add a user to your organization" style="width:80%;">}}
-
-The new user receives an email with a link to log in. The user is marked with status `Invite Pending` until they log in. To cancel their invite before they log in, click the *Delete Invite* button on the right of the user line in list view, or on the user box in grid view. 
-
-{{< img src="account_management/users/delete_invite_grid.png" alt="Delete invite on the grid view" style="width:50%;">}}
+5. Click **Send Invites**.
 
-To resend an invite in list view, click the user to open the user side panel, and click **Resend Invite**:
+The new user receives an email with a link to log in. The user is marked with the status `Invite Pending` until they log in. To cancel their invite before they log in, click the *Delete Invite* button on the right of the user line in list view, or on the user box in grid view.
```
Comment (alai97) [inline content/en/account_management/users/_index.md:38]:
```suggestion
The new user receives an email with a link to log in. The user is marked with the status `Invite Pending` until they log in. To cancel their invite before they log in, click the **Delete Invite** button on the right of the user line in list view, or on the user box in grid view. 
```

File: content/en/account_management/plan_and_usage.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+## Plan settings
+
+Upgrade or downgrade your account from the Plan tab on the Plan & Usage section.
+
+**Note**: Contact your account representative or [success@datadoghq.com][4] to add or remove paid features that are not part of standard plans.
+
+
+## Billing history
+
+Your account billing history is available from the Billing History tab in the [Plan & Usage section][2]. For specific billing questions, contact your account representative or [success@datadoghq.com][4].
```
Comment (alai97) [inline content/en/account_management/plan_and_usage.md:25]:
```suggestion
Your account billing history is available from the **Billing History** tab in the [Plan & Usage section][2]. For specific billing questions, contact your account representative or [success@datadoghq.com][4].
```

File: content/en/account_management/plan_and_usage.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+View hourly usage of your account across all billable parameters as well as the top 500 custom metrics submitted. If your organization is the parent-organization in a [multi-org account][3], you can also view the usage of child-organization accounts by toggling between "Overall" and "Individual Organizations" at the top of the page. Individual child-organization accounts are isolated. They are only able to view their own usage and custom metrics.
+
+**Note**: Allow up to 72 hours for this section to update.
+
+Further detailed usage reports can also be created [on request][4] by your account representative.
+
+## Plan settings
+
+Upgrade or downgrade your account from the Plan tab on the Plan & Usage section.
```
Comment (alai97) [inline content/en/account_management/plan_and_usage.md:18]:
```suggestion
Upgrade or downgrade your account from the **Plan** tab on the Plan & Usage section.
```

File: content/en/account_management/org_settings.md [hunk 9]
```diff
@@ -2,38 +2,77 @@
...
+
+The **Audeit Logs** tab in the Organization Settings page opens a new tab to the Audit Events Explorer.
+#### Audit Log Settings
+
+The **Audit Log Settings** tab allows you to set the retention period of audit logs, and enable archiving to other cloud storage services.
+### General
+
+#### Rename organization
+
+To rename your organization, click the **Rename organization** tab, enter the new name, then click save.
```
Comment (alai97) [inline content/en/account_management/org_settings.md:67]:
```suggestion
To rename your organization, click the **Rename organization** tab, enter the new name, then click **Save**.
```

File: content/en/account_management/org_settings.md [hunk 10]
```diff
@@ -2,38 +2,77 @@
...
+
+**Note**: **Your organization name may not exceed 32 characters**.
```
Comment (alai97) [inline content/en/account_management/org_settings.md:69]:
```suggestion
**Note: Your organization name may not exceed 32 characters.**
```

File: content/en/account_management/org_settings.md [hunk 11]
```diff
@@ -2,38 +2,77 @@
...
+### Security
+
+#### Public Sharing
+
+The **Public Sharing** tab contains lists of shared dashboards and shared graphs. You can also edit your sharing settings by clicking the **Enabled** toggles.
+
+#### Authentication
+
+The **Authentication** tab shows password, Google, and SAML authentication settings. You can toggle each with the **Enabled by Default** dropdowns.
+#### Audit Logs
```
Comment (alai97) [inline content/en/account_management/org_settings.md:57]:
```suggestion

#### Audit Logs
```

File: content/en/account_management/org_settings.md [hunk 12]
```diff
@@ -2,38 +2,77 @@
...
+
+The **Audeit Logs** tab in the Organization Settings page opens a new tab to the Audit Events Explorer.
+#### Audit Log Settings
```
Comment (alai97) [inline content/en/account_management/org_settings.md:60]:
```suggestion

#### Audit Log Settings
```

File: content/en/account_management/org_settings.md [hunk 13]
```diff
@@ -2,38 +2,77 @@
...
+
+The **Audit Log Settings** tab allows you to set the retention period of audit logs, and enable archiving to other cloud storage services.
+### General
```
Comment (alai97) [inline content/en/account_management/org_settings.md:63]:
```suggestion

### General
```

File: content/en/account_management/org_settings.md [hunk 14]
```diff
@@ -2,38 +2,77 @@
...
+
+#### Authentication
+
+The **Authentication** tab shows password, Google, and SAML authentication settings. You can toggle each with the **Enabled by Default** dropdowns.
+#### Audit Logs
+
+The **Audeit Logs** tab in the Organization Settings page opens a new tab to the Audit Events Explorer.
+#### Audit Log Settings
+
+The **Audit Log Settings** tab allows you to set the retention period of audit logs, and enable archiving to other cloud storage services.
```
Comment (alai97) [inline content/en/account_management/org_settings.md:62]:
```suggestion
The **Audit Log Settings** tab allows you to set the retention period of audit logs and enable archiving to other cloud storage services.
```

File: content/en/account_management/org_settings.md [hunk 15]
```diff
@@ -2,38 +2,77 @@
...
+You can filter application keys by name, ID, or owner, or click the **Only My Keys** toggle to only view application keys you own. Read the [documentation on application keys][6] for more information on how to add or remove them.
 
-For more information about configuring SAML, reference the [Single sign on with SAML documentation][5].
+#### Client Tokens
+
+Client tokens are used to send events and logs from your user’s web and mobile applications. They are unique to your organization. Deleting a client token that is linked to a RUM Application causes your RUM Application to stop reporting. The [process to create them][7] is similar to API and Application Keys.
+
+#### Events API Emails
+If your application does not have an existing Datadog integration, and you don’t want to create a custom Agent check, you can send events with email. Read the guide on [events with email][8] to learn how to set up events API emails.
+
```
Comment (alai97) [inline content/en/account_management/org_settings.md:46]:
```suggestion
If your application does not have an existing Datadog integration, and you don’t want to create a custom Agent check, you can send events with email. To learn how to set up events API emails, read the [Events with email guide][8].
```

File: content/en/account_management/org_settings.md [hunk 16]
```diff
@@ -2,38 +2,77 @@
...
 
-## Change your organization name
+This section allows you to view, copy, and revoke any API key in the list. Your API Keys are unique to your organization. An API key is required by the Datadog Agent to submit metrics and events to Datadog. Read the [API keys documentation][6] for more information on creating, editing, and revoking keys.
 
-To rename your organization, enter the new name, then click save.
-**Note**: **Your organization name may not exceed 32 characters**.
+#### Application Keys
 
-## SAML settings
+You can filter application keys by name, ID, or owner, or click the **Only My Keys** toggle to only view application keys you own. Read the [documentation on application keys][6] for more information on how to add or remove them.
```
Comment (alai97) [inline content/en/account_management/org_settings.md:38]:
```suggestion
You can filter application keys by name, ID, or owner, or click the **Only My Keys** toggle to only view application keys you own. Read the [Application keys documentation][6] for more information on how to add or remove them.
```

File: content/en/account_management/org_settings.md [hunk 17]
```diff
@@ -2,38 +2,77 @@
...
 
-## Usage reports
+Read the [Role Based Access Controll][3] documentation to learn about default and custom roles in Datadog.
+#### SAML Group Mappings
 
-View hourly usage of your account across all billable parameters as well as the top 500 custom metrics submitted. If your organization is the parent-organization in a [multi-org account][4], you can also view the usage of child-organization accounts. Individual child-organization accounts are isolated. They are only able to view their own usage and custom metrics.
+When enabled, users logging in with SAML to your Datadog account are stripped of their current roles and reassigned to new roles based on the details in their SAML assertion passed on from your Identity Provider, and the mappings you've created.
 
-**Note**: Allow up to 72 hours for this section to update.
+Users who log in with SAML and do not have the values that map to a Datadog role will be stripped of all roles and are not allowed to log in.
```
Comment (alai97) [inline content/en/account_management/org_settings.md:24]:
```suggestion
Users who log in with SAML and do not have the values that map to a Datadog role are stripped of all roles and are not allowed to log in.
```

File: content/en/account_management/org_settings.md [hunk 18]
```diff
@@ -2,38 +2,77 @@
...
+Read the [Mapping SAML attributes][4] documentation to learn more about how to create and set mappings.
```
Comment (alai97) [inline content/en/account_management/org_settings.md:25]:
```suggestion
To learn how to create and set mappings, read the [Mapping SAML attributes documentation][4].
```

File: content/en/account_management/org_settings.md [hunk 19]
```diff
@@ -2,38 +2,77 @@
...
 
-**Note**: Contact your account representative or [success@datadoghq.com][3] to add or remove paid features that are not part of standard plans.
+#### Roles
 
-## Usage reports
+Read the [Role Based Access Controll][3] documentation to learn about default and custom roles in Datadog.
+#### SAML Group Mappings
 
-View hourly usage of your account across all billable parameters as well as the top 500 custom metrics submitted. If your organization is the parent-organization in a [multi-org account][4], you can also view the usage of child-organization accounts. Individual child-organization accounts are isolated. They are only able to view their own usage and custom metrics.
+When enabled, users logging in with SAML to your Datadog account are stripped of their current roles and reassigned to new roles based on the details in their SAML assertion passed on from your Identity Provider, and the mappings you've created.
```
Comment (alai97) [inline content/en/account_management/org_settings.md:22]:
```suggestion
When enabled, users logging in with SAML to your Datadog account are stripped of their current roles and reassigned to new roles based on the details in their SAML assertion passed on from your Identity Provider and the mappings you've created.
```

File: content/en/account_management/org_settings.md [hunk 20]
```diff
@@ -2,38 +2,77 @@
...
+#### Service Accounts
 
-Upgrade or downgrade your account from the Plan tab on the Organization Account Settings page.
+### Groups
 
-**Note**: Contact your account representative or [success@datadoghq.com][3] to add or remove paid features that are not part of standard plans.
+#### Roles
 
-## Usage reports
+Read the [Role Based Access Controll][3] documentation to learn about default and custom roles in Datadog.
```
Comment (alai97) [inline content/en/account_management/org_settings.md:19]:
```suggestion
To learn about default and custom roles in Datadog, read the [Role Based Access Control documentation][3].
```

File: content/en/account_management/org_settings.md [hunk 21]
```diff
@@ -2,38 +2,77 @@
 title: Organization Settings
 kind: documentation
 ---
+## Overview
+The Organization Settings section is available to [Administrators][1] by clicking **Organization Settings** from the account menu in the bottom of the left side navigation or by selecting **Organization Settings** from the header dropdown at the top of the Personal Settings page.
 
-[Administrators][1] can access the [organization account settings][2] by clicking Plan & Usage from the Account menu in the left side navigation or from the "Go to organization settings" button at the top of the Account Settings page.
+The Organization Settings allow you to manage users, groups, RBAC, keys and tokens, more. This page outlines each section and where in the documentation you can learn about specific tasks in **Organization Settings**.
```
Comment (alai97) [inline content/en/account_management/org_settings.md:8]:
```suggestion
Organization Settings allow you to manage users, groups, RBAC, keys, and tokens. This page outlines every section and where in the documentation you can learn about specific tasks in **Organization Settings**.
```

File: content/en/account_management/api-app-keys.md [hunk 22]
```diff
@@ -17,44 +17,38 @@ API keys are unique to your organization. An [API key][1] is required by the Dat
...
-
-* Application key names cannot be blank.
+**Note**: Application key names cannot be blank.
 
 ## Remove application keys
 
-To remove a Datadog application key, navigate to [Teams -> Application Keys][2]. If you have the [permission][5] to create and manage application keys, you can see your own keys and click **Revoke** next to the key you want to revoke. If you have the permission to manage all org application keys, you can search for the key you want to revoke and then click **Revoke** next to it:
-
-{{< img src="account_management/api_app_keys/application_keys_revoke.png" alt="Revoke Application Keys"  >}}
+To remove a Datadog application key, navigate to **Organization Settings -> Application Keys**. If you have the [permission][5] to create and manage application keys, you can see your own keys and click **Revoke** next to the key you want to revoke. If you have the permission to manage all org application keys, you can search for the key you want to revoke and then click **Revoke** next to it.
```
Comment (alai97) [inline content/en/account_management/api-app-keys.md:51]:
```suggestion
To remove a Datadog application key, navigate to **Organization Settings** > **Application Keys**. If you have the [permission][5] to create and manage application keys, you can see your own keys and click **Revoke** next to the key you want to revoke. If you have the permission to manage all org application keys, you can search for the key you want to revoke and click **Revoke** next to it.
```

File: content/en/account_management/api-app-keys.md [hunk 23]
```diff
@@ -17,44 +17,38 @@ API keys are unique to your organization. An [API key][1] is required by the Dat
...
 
-To add a Datadog application key, navigate to [Teams -> Application Keys][2]. If you have the [permission][5] to create application keys you can click **New Key**:
-
-{{< img src="account_management/api_app_keys/application_keys_new_key.png" alt="Create new Application Keys"  >}}
+To add a Datadog application key, navigate to **Organization Settings -> Application Keys**. If you have the [permission][5] to create application keys you can click **New Key**.
 
-**Note**:
-
-* Application key names cannot be blank.
+**Note**: Application key names cannot be blank.
```
Comment (alai97) [inline content/en/account_management/api-app-keys.md:47]:
```suggestion
**Note:** Application key names cannot be blank.
```

File: content/en/account_management/api-app-keys.md [hunk 24]
```diff
@@ -17,44 +17,38 @@ API keys are unique to your organization. An [API key][1] is required by the Dat
...
-
-{{< img src="account_management/api_app_keys/api_keys_revoke.png" alt="Revoke API Keys"  >}}
+To remove a Datadog API key or client token, navigate to the list of keys or tokens, and click the trash can icon with **Revoke** next to the key or token you want to remove.
 
 ## Add application keys
 
-To add a Datadog application key, navigate to [Teams -> Application Keys][2]. If you have the [permission][5] to create application keys you can click **New Key**:
-
-{{< img src="account_management/api_app_keys/application_keys_new_key.png" alt="Create new Application Keys"  >}}
+To add a Datadog application key, navigate to **Organization Settings -> Application Keys**. If you have the [permission][5] to create application keys you can click **New Key**.
```
Comment (alai97) [inline content/en/account_management/api-app-keys.md:45]:
```suggestion
To add a Datadog application key, navigate to **Organization Settings** > **Application Keys**. If you have the [permission][5] to create application keys, click **New Key**.
```

File: content/en/account_management/api-app-keys.md [hunk 25]
```diff
@@ -17,44 +17,38 @@ API keys are unique to your organization. An [API key][1] is required by the Dat
...
 * Your org must have at least one API key and at most 50 API keys.
-* Key names must be unique across your org.
+* Key names must be unique across your organization.
 
 ## Remove API keys or client tokens
 
-To remove a Datadog API key or client token, navigate to [Integration -> APIs][1] and click **Revoke** next to the key or token you want to remove:
-
-{{< img src="account_management/api_app_keys/api_keys_revoke.png" alt="Revoke API Keys"  >}}
+To remove a Datadog API key or client token, navigate to the list of keys or tokens, and click the trash can icon with **Revoke** next to the key or token you want to remove.
```
Comment (alai97) [inline content/en/account_management/api-app-keys.md:41]:
```suggestion
To remove a Datadog API key or client token, navigate to the list of keys or tokens, and click the **trash can** icon with **Revoke** next to the key or token you want to remove.
```

File: content/en/account_management/api-app-keys.md [hunk 26]
```diff
@@ -17,44 +17,38 @@ API keys are unique to your organization. An [API key][1] is required by the Dat
...
 ## Add an API key or client token
 
-To add a Datadog API key or client token, navigate to [Integration -> APIs][1], enter a name for your key or token, and click **Create API key** or **Create Client Token**.
+To add a Datadog API key or client token:
+1. Navigate to Organization settings, then click the **API keys** or **Client Tokens** tab.
+3. Click the **New Key** or **New Client Token** button, depending on which you're creating.
+4. Enter a name for your key or token.
+4. Click **Create API key** or **Create Client Token**.
 
 **Note**:
```
Comment (alai97) [inline content/en/account_management/api-app-keys.md:34]:
```suggestion
**Notes:**
```

File: content/en/account_management/api-app-keys.md [hunk 27]
```diff
@@ -17,44 +17,38 @@ API keys are unique to your organization. An [API key][1] is required by the Dat
 
 ## Client tokens
 
-To manage your client tokens, go to your [Datadog API configuration page][1] in the `Client Tokens` section as shown here:
-
-{{< img src="account_management/api_app_keys/client_tokens.png" style="width:80%;" alt="Client tokens"  >}}
+To manage your client tokens, go to Organization settings, then click the **Client Tokens** tab.
```
Comment (alai97) [inline content/en/account_management/api-app-keys.md:20]:
```suggestion
To manage your client tokens, go to **Organization Settings**, then click the **Client Tokens** tab.
```

File: content/en/account_management/_index.md [hunk 28]
```diff
@@ -38,40 +49,30 @@ If you are an organization administrator, reference the additional documentation
...
-You can set your timezone, desktop notifications, and email subscriptions from the [*Preferences* tab][9] within the *account settings* page. Under email subscriptions, you have access to the following reports:
-
-* Daily Digest
-* Weekly Digest
-* Weekly Monitor Report
-* Weekly Pagerduty Report
-* Weekly Nagios Report
+### Security
 
+The *Application Keys* tab in *Personal Settings* allows you to manage your application keys. To copy a key, hover over it until the **Copy Key** icon appears to the right, and click on it. You can also click into a specific key to edit its name, view when it was created, view the profile of the key's owner, copy it, or revoke it.
```
Comment (alai97) [inline content/en/account_management/_index.md:54]:
```suggestion
The **Application Keys** tab in **Personal Settings** allows you to manage your application keys. To copy a key, hover over it until the **Copy Key** icon appears to the right, and click on it. You can also click into a specific key to edit its name, view when it was created, view the profile of the key's owner, copy it, or revoke it.
```

File: content/en/account_management/_index.md [hunk 29]
```diff
@@ -10,21 +10,32 @@ aliases:
...
+* Weekly Digest
+* Weekly Monitor Report
+* Weekly Pagerduty Report
+* Weekly Nagios Report
+
+If you are unsure if an email digest or report is relevant to you, you can view an example by clicking the **Example** link next to each email subscription. You can also use the **Unsubscribe From All** button to quickly unsubscribe from all email subscriptions.
 ### Organizations
 
-The *account settings* page also lists all of the organizations you are associated with. Switch between these organizations from this page or by hovering over the account menu in the left-side navigation.
+The *Organizations* page in *Personal Settings* lists all of the organizations you are associated with. Switch between these organizations from this page or by hovering over the account menu in the left-side navigation.
```
Comment (alai97) [inline content/en/account_management/_index.md:38]:
```suggestion
The **Organizations** page in **Personal Settings** lists all of the organizations you are associated with. Switch between these organizations from this page or by hovering over the account menu in the left-side navigation.
```

File: content/en/account_management/_index.md [hunk 30]
```diff
@@ -10,21 +10,32 @@ aliases:
...
+
+You can set your timezone, desktop notifications, and email subscriptions from the [*Preferences* tab][3] within the *Personal Settings* page. Under email subscriptions, you have access to the following reports:
+
+* Daily Digest
+* Weekly Digest
+* Weekly Monitor Report
+* Weekly Pagerduty Report
+* Weekly Nagios Report
+
+If you are unsure if an email digest or report is relevant to you, you can view an example by clicking the **Example** link next to each email subscription. You can also use the **Unsubscribe From All** button to quickly unsubscribe from all email subscriptions.
```
Comment (alai97) [inline content/en/account_management/_index.md:35]:
```suggestion
If you are unsure if an email digest or report is relevant to you, view an example by clicking the **Example** link next to each email subscription. You can also use the **Unsubscribe From All** button to quickly unsubscribe from all email subscriptions.

```

File: content/en/account_management/_index.md [hunk 31]
```diff
@@ -10,21 +10,32 @@ aliases:
...
 
-To update your picture, create an account on [Gravatar][2] and associate it with your email address.
+To update your picture, create an account on [Gravatar][1] and associate it with your email address.
 
-If you log in to Datadog using Google authentication, your email address is provided by your Google account and is *not* editable within Datadog. To change your email address in Google, see the [Google documentation][3].
+If you log in to Datadog using Google authentication, your email address is provided by your Google account and is *not* editable within Datadog. To change your email address in Google, see the [Google documentation][2].
 
+### Preferences
+
+You can set your timezone, desktop notifications, and email subscriptions from the [*Preferences* tab][3] within the *Personal Settings* page. Under email subscriptions, you have access to the following reports:
```
Comment (alai97) [inline content/en/account_management/_index.md:27]:
```suggestion
You can set your timezone, desktop notifications, and email subscriptions from the [**Preferences** tab][3] within the **Personal Settings** page. Under email subscriptions, you have access to the following reports:
```

File: content/en/account_management/_index.md [hunk 32]
```diff
@@ -10,21 +10,32 @@ aliases:
...
 ### Profile
 
-Your profile is how others in your organization recognize you in Datadog. Set or update your name, email address and organizational role here.
+Your profile is how others in your organization recognize you in Datadog. Set or update your name, email address and title here.
 
-To update your picture, create an account on [Gravatar][2] and associate it with your email address.
+To update your picture, create an account on [Gravatar][1] and associate it with your email address.
 
-If you log in to Datadog using Google authentication, your email address is provided by your Google account and is *not* editable within Datadog. To change your email address in Google, see the [Google documentation][3].
+If you log in to Datadog using Google authentication, your email address is provided by your Google account and is *not* editable within Datadog. To change your email address in Google, see the [Google documentation][2].
```
Comment (alai97) [inline content/en/account_management/_index.md:23]:
```suggestion
If you log in to Datadog using Google authentication, your email address is provided by your Google account and is **not** editable within Datadog. To change your email address in Google, see the [Google documentation][2].
```

File: content/en/account_management/_index.md [hunk 33]
```diff
@@ -10,21 +10,32 @@ aliases:
...
-## Account settings
+## Personal settings
 
-The *[account settings][1]* page in Datadog allows you to control how you appear to others in your organization; switch or leave organizations; manage your notification preferences and more.
+The personal settings pages in Datadog allow you to control how you appear to others in your organization; switch or leave organizations; manage your notification preferences and more.
 
 ### Profile
 
-Your profile is how others in your organization recognize you in Datadog. Set or update your name, email address and organizational role here.
+Your profile is how others in your organization recognize you in Datadog. Set or update your name, email address and title here.
```
Comment (alai97) [inline content/en/account_management/_index.md:19]:
```suggestion
Your profile is how others in your organization recognize you in Datadog. Set or update your name, email address, and title here.
```

File: content/en/account_management/_index.md [hunk 34]
```diff
@@ -10,21 +10,32 @@ aliases:
 <div class="alert alert-warning">The Datadog for Government site only supports SAML login.</div>
 {{< /site-region >}}
 
-## Account settings
+## Personal settings
 
-The *[account settings][1]* page in Datadog allows you to control how you appear to others in your organization; switch or leave organizations; manage your notification preferences and more.
+The personal settings pages in Datadog allow you to control how you appear to others in your organization; switch or leave organizations; manage your notification preferences and more.
```
Comment (alai97) [inline content/en/account_management/_index.md:15]:
```suggestion
The personal settings pages in Datadog allow you to control how you appear to others in your organization, switch or leave organizations, and manage your notification preferences.
```

**Review by alai97 — CHANGES_REQUESTED** [review]
Mostly bold and spacing suggestions!

**Review by alai97 — APPROVED** [review]

**Pull request merged**